build error on ubuntu 5.11.0-1007-raspi

build message:
In file included from /home/konvekta/devel/OpcUaStack/src/OpcUaStackCore/Certificate/CertificateSettings.cpp:18:
/home/konvekta/devel/OpcUaStack/src/OpcUaStackCore/Certificate/CertificateSettings.h:72:15: error: field ‘serverUri_’ has incomplete type ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’}
   72 |   std::string serverUri_;
      |               ^~~~~~~~~~
In file included from /usr/include/c++/10/iosfwd:39,
                 from /usr/include/c++/10/memory:74,
                 from /usr/include/boost/smart_ptr/detail/sp_counted_impl.hpp:35,
                 from /usr/include/boost/smart_ptr/detail/shared_count.hpp:27,
                 from /usr/include/boost/smart_ptr/shared_ptr.hpp:17,
                 from /usr/include/boost/shared_ptr.hpp:17,
                 from /home/konvekta/devel/OpcUaStack/src/OpcUaStackCore/Certificate/CertificateSettings.h:22,
                 from /home/konvekta/devel/OpcUaStack/src/OpcUaStackCore/Certificate/CertificateSettings.cpp:18:
/usr/include/c++/10/bits/stringfwd.h:74:11: note: declaration of ‘std::string’ {aka ‘class std::__cxx11::basic_string<char>’}
   74 |     class basic_string;
      |           ^~~~~~~~~~~~

--- a/src/OpcUaStackCore/Certificate/CertificateSettings.h
+++ b/src/OpcUaStackCore/Certificate/CertificateSettings.h
@@ -1,5 +1,5 @@
 /*
-   Copyright 2019 Kai Huebl (kai@huebl-sgh.de)
+   Copyright 2019-2021 Kai Huebl (kai@huebl-sgh.de)
 
    Lizenziert gemäß Apache Licence Version 2.0 (die „Lizenz“); Nutzung dieser
    Datei nur in Übereinstimmung mit der Lizenz erlaubt.
@@ -19,6 +19,7 @@
 #define __OpcUaStackCore_CertificateSettings_h__
 
 #include <vector>
+#include <string>
 #include <boost/shared_ptr.hpp>
 #include "OpcUaStackCore/Base/os.h"
 

